packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -643,20 +643,24 @@ const ComboboxInner = forwardRef(function ComboboxInner(props: ComboBoxProps<any
           description={descriptionMessage}>
           {errorMessage}
         </HelpText>
-        <PopoverBase
+        <Popover
           hideArrow
           triggerRef={triggerRef}
           offset={menuOffset}
           placement={`${direction} ${align}` as Placement}
           shouldFlip={shouldFlip}
           UNSAFE_style={{
-            width: menuWidth ? `${menuWidth}px` : undefined,
-            // manually subtract border as we can't set Popover to border-box, it causes the contents to spill out
-            '--trigger-width': `calc(${triggerWidth} - 2px)`
+            // Subtract by 2 since these widths are set on the inner div rather than on the outermost div element that has
+            // a border
+            width: menuWidth ? `calc(${menuWidth}px - 2 * var(--s2-container-border-width))` : undefined,
```

Comment:
should this just be on the --trigger-width variable? i'm not sure why width is an inline style AND a css class property

Reply:
good point, was too focused on replacing things in place